web/src/user_card_popover.ts

```diff
@@ -283,6 +283,9 @@ function get_user_card_popover_data(
     const can_send_private_message =
         user_can_send_direct_message(user_id_string) && is_active && !is_me;
 
+    const user_last_seen_time_status =
+        buddy_data.get_user_last_seen_status(user.user_id) ?? $t({defaultMessage: "Loading..."});
```

Comment:
`Loading…`, with the unicode elipses, looks better and is what we use in the codebase.